test/libsolidity/InlineAssembly.cpp

```diff
@@ -255,8 +296,8 @@ BOOST_AUTO_TEST_CASE(assignment_after_tag)
 
 BOOST_AUTO_TEST_CASE(magic_variables)
 {
-	BOOST_CHECK(!successAssemble("{ this }"));
-	BOOST_CHECK(!successAssemble("{ ecrecover }"));
+	CHECK_ASSEMBLE_ERROR("{ this pop }", DeclarationError, "Identifier not found or not unique");
```

Comment:
The code above is `parseAndReturnFirstError`. In this case is the stack height the first error returned? If not we do not need the pop?